src/main.cpp

```diff
@@ -175,26 +262,8 @@ int main ( int argc, char** argv )
             continue;
         }
 
-        // JSON-RPC port number ------------------------------------------------
-        if ( GetNumericArgument ( argc, argv, i, "--jsonrpcport", "--jsonrpcport", 0, 65535, rDbleArgument ) )
-        {
-            iJsonRpcPortNumber = static_cast<quint16> ( rDbleArgument );
-            qInfo() << qUtf8Printable ( QString ( "- JSON-RPC port number: %1" ).arg ( iJsonRpcPortNumber ) );
-            CommandLineOptions << "--jsonrpcport";
-            continue;
-        }
-
-        // JSON-RPC secret file name -------------------------------------------
-        if ( GetStringArgument ( argc, argv, i, "--jsonrpcsecretfile", "--jsonrpcsecretfile", strArgument ) )
-        {
-            strJsonRpcSecretFileName = strArgument;
-            qInfo() << qUtf8Printable ( QString ( "- JSON-RPC secret file: %1" ).arg ( strJsonRpcSecretFileName ) );
-            CommandLineOptions << "--jsonrpcsecretfile";
-            continue;
-        }
-
```

Comment:
Why did that get removed?

Reply:
Good point !!
I have no idea how that happened !